modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, you should:
+
+* Have access to the cluster as a user with `cluster-admin` privileges.
```

Comment:
This can probably be removed since it's already a prereq in all the modules that need it (though I'd double check whether it's also a necessary prereq for the module about changing the update server)